src/hotspot/cpu/x86/x86.ad

```diff
@@ -1649,6 +1649,9 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
       break;
     case Op_RotateRightV:
     case Op_RotateLeftV:
+      if (is_subword_type(bt)) {
```

Comment:
Does that have the effect of not intrinsifying for `byte` or `short`?

Reply:
Yes, it makes sure that intrinsification is based on Shifts and Or operations instead of Rotation operation.